templates/image.jsx

```diff
@@ -34,7 +34,10 @@ export default function Image(props) {
     >
 
       <img
-        className={prefixClasses(props.classNamePrefixes, ['__image'])}
+        className={classes([
+          prefixClasses(props.classNamePrefixes, ['__image']),
+          props?._srcFocalPoint && `object-position-${props?._srcFocalPoint}`
```

Comment:
```suggestion
          props._srcFocalPoint && `object-position-${props._srcFocalPoint}`
```
@guywillis We shouldn't need the optional chaining (?.) here since React will just ignore it if the property is undefined. No risk of throwing an error.

Also, wondering if this property should drop the underscore and just use `srcFocalPoint`? Other properties that get passed to this template like `classNamePrefixes` are also not translatable and don't use an underscore. So, that would make it consistent.

Reply:
`_srcFocalPoint` will often be supplied straight from the json,  where it would require the `_` as it is not translatable, removing the `_` from here would require code to remove the underscore from the json attribute.

`classNamePrefixes` is a code supplied property, usually of the component utilising the behaviour, `hotgraphic`, `accordion` etc, not a json supplied property.